compute/apiv1/smoke_test.go

```
@@ -0,0 +1,290 @@
+
+	rules := []*computepb.SecurityPolicyRule{
+		securityPolicyRule,
+		securityPolicyRuleDefault,
+	}
+
+	securityPolicy := &computepb.SecurityPolicy{
+		Name: &name,
+		Rules: rules,
+	}
+
+	insertRequest := &computepb.InsertSecurityPolicyRequest{
+		Project: projectId,
+		SecurityPolicyResource: securityPolicy,
+	}
+	insert, err := c.Insert(ctx, insertRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+
+	waitGlobalRequest := &computepb.WaitGlobalOperationRequest{
+		Project: projectId,
+		Operation: insert.GetName(),
+	}
+	_, err = globalCLient.Wait(ctx, waitGlobalRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+	fmt.Printf("Inserted security policy named %s\n", name)
+	defer ForceDeleteSecurityPolicy(name, ctx, c)
+
+	removeRuleRequest := &computepb.RemoveRuleSecurityPolicyRequest{
+		Priority: &priority,
+		Project: projectId,
+		SecurityPolicy: name,
+	}
+
+	rule, err := c.RemoveRule(ctx, removeRuleRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+	waitGlobalRequestRemove := &computepb.WaitGlobalOperationRequest{
+		Project: projectId,
+		Operation: rule.GetName(),
+	}
+	_, err = globalCLient.Wait(ctx, waitGlobalRequestRemove)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+
+	getRequest := &computepb.GetSecurityPolicyRequest{
+		Project: projectId,
+		SecurityPolicy: name,
+	}
+	get, err := c.Get(ctx, getRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+	assert.Equal(t, 1, len(get.GetRules()))
+
+	deleteRequest := &computepb.DeleteSecurityPolicyRequest{
+		Project: projectId,
+		SecurityPolicy: name,
+	}
+	_, err = c.Delete(ctx, deleteRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+}
+
+func ForceDeleteSecurityPolicy(name string, ctx context.Context, client *SecurityPoliciesClient){
+	deleteRequest := &computepb.DeleteSecurityPolicyRequest{
+		Project: projectId,
+		SecurityPolicy: name,
+	}
+	_, err := client.Delete(ctx, deleteRequest)
+	if err != nil {}
```

Empty conditional. Should this be a t.Helper?

---

I removed empty conditional, we want to swallow error here